xbmc/interfaces/legacy/Window.h

```diff
@@ -655,81 +655,6 @@ namespace XBMCAddon
       SWIGHIDDENVIRTUAL long getWidth();
 #endif
 
-#ifdef DOXYGEN_SHOULD_USE_THIS
-      ///
-      /// \ingroup python_xbmcgui_window
-      /// @brief \python_func{ getResolution() }
-      ///-----------------------------------------------------------------------
-      /// Returns The resolution of the screen
-      ///
-      /// @return                       Used Resolution
-      ///  The returned value is one of the following:
-      ///  | value | Resolution                |
-      ///  |:-----:|:--------------------------|
-      ///  |   0   | 1080i      (1920x1080)
-      ///  |   1   | 720p       (1280x720)
-      ///  |   2   | 480p 4:3   (720x480)
-      ///  |   3   | 480p 16:9  (720x480)
-      ///  |   4   | NTSC 4:3   (720x480)
-      ///  |   5   | NTSC 16:9  (720x480)
-      ///  |   6   | PAL 4:3    (720x576)
-      ///  |   7   | PAL 16:9   (720x576)
-      ///  |   8   | PAL60 4:3  (720x480)
-      ///  |   9   | PAL60 16:9 (720x480)
-      ///
-      ///-----------------------------------------------------------------------
-      /// @python_v18 Deprecated.
```

Comment:
this was deprecated but we didn't get any alternative. Is there any? I'm not familiar with this method. Also need python_removed_functions

Reply:
i believe it's `xbmcgui.Window.getWidth()` and `xbmcgui.Window.getHeight()`
see: https://github.com/xbmc/xbmc/pull/12279

perhaps @romanvm can confirm

Reply:
Yes seems right. Since this whole block is going away, and since we probably will have yet more leia releases we should probably fix this deprecation note (and merge it before this PR goes in) so that we can backport it to leia before the block is completely removed.

Reply:
Reading about this a bit better getWidth and getHeight are not direct replacements for the removed methods. Nothing to do here.

Reply:
This method is probably a remnant from the old Xbox days with its limited set of screen resolutions (hence the limited set of numeric constants) and does not provide any useful info for modern systems/displays. I have added `xbmcgui.getScreenWidth()`/`xbmcgui.getScreenHeight()` functions to get a real display resolution from Python.